ballerina-tests/http-misc-tests/tests/http_payload_test.bal

```diff
@@ -116,8 +116,9 @@ function testGetXmlPayloadReturnParserError() returns error? {
     if response is http:Response {
         test:assertEquals(response.statusCode, 200, msg = "Found unexpected output");
         common:assertHeaderValue(check response.getHeader(common:CONTENT_TYPE), common:TEXT_PLAIN);
-        common:assertTrueTextPayload(response.getTextPayload(),
-                "Error occurred while extracting xml data from entity: error(\"failed to create xml");
+        common:assertTrueTextPayloadWithMutipleOptions(response.getTextPayload(),
+                "Error occurred while extracting xml data from entity: error(\"failed to create xml",
```

Comment:
This will be removed and additional changes with be removed once https://github.com/ballerina-platform/ballerina-lang/pull/40286 is merged.